examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -8,8 +8,14 @@ authToken: ""
 
 # Address of the teleport proxy with port (usually :3080).
 proxyAddr: ""
-# Comma-separated list of roles to enable (any of: kube,db,app)
-roles: "kube"
+# Comma-separated list of roles to enable (any of: kube,db,app,discovery)
+roles: "kube,app,discovery"
```

Comment:
We can't change this default value otherwise we will break the existing configuration